docs/reference/configuration/Drivers.md

```diff
@@ -1,6 +1,6 @@
 <h2 id="configuration-drivers">Drivers</h2>
 
-The driver configurations allows for customization of the OS driver layer. Options in this section pertain to hardware IO. 
+The driver configurations allows for customization of the OS driver layer. Options in this section pertain to hardware IO. You can find additional serial settings for `printf` in the [platform page](platform.html). These settings do not affect Serial objects.
```

Comment:
Note: This currently links to the Platform overview page of the user API section. Is that correct, or did you want to link to the platform config page (configuration-platform.html)?

Reply:
Nice catch, configuration-platform is correct.